### PR TITLE
Control snapshot chunk sending traffic

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Control snapshot chunk sending traffic (i.e., send 1 chunk per second) 
- Reject snapshot chunk request when handling another request

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/713

### CODE REVIEW

File: node/index.js [hunk 1]
```diff
@@ -337,7 +390,7 @@ class BlockchainNode {
     logger.info(`[${LOG_HEADER}] Now node in CHAIN_SYNCING state!`);
 
     // 6. Reset latest snapshot.
```
Comment (liayoo) [inline node/index.js:392]: ditto

File: node/index.js [hunk 2]
```diff
@@ -59,9 +59,11 @@ class BlockchainNode {
     this.db = DB.create(
         StateVersions.EMPTY, initialVersion, this.bc, false, this.bc.lastBlockNumber(),
         this.stateManager, eventHandler);
-    this.latestSnapshotSource = null;
-    this.latestSnapshot = null;
-    this.latestSnapshotBlockNumber = -1;
+    this.preparedSnapshotSource = null;
```
Comment (liayoo) [inline node/index.js:62]: suggestion) what do you think about changing preparedSnapshot* to bootstrapSnapshot*?

File: node/index.js [hunk 3]
```diff
@@ -298,30 +326,55 @@ class BlockchainNode {
...
-        `[${LOG_HEADER}] Initializing DB with latest snapshot from ${this.latestSnapshotSource}..`);
+        `[${LOG_HEADER}] Initializing DB with latest snapshot from ${this.preparedSnapshotSource}..`);
     const startingDb = DB.create(
-        StateVersions.EMPTY, StateVersions.START, this.bc, true, this.latestSnapshotBlockNumber,
+        StateVersions.EMPTY, StateVersions.START, this.bc, true, this.preparedSnapshotBlockNumber,
         this.stateManager);
-    startingDb.initDb(this.latestSnapshot);
+    startingDb.initDb(this.preparedSnapshot);
 
     // 2. Initialize the blockchain, starting from `latestSnapshotBlockNumber`.
```
[the remark this answers is not in the document]
Comment (platfowner) [reply node/index.js]: Updated, thanks!

File: node/index.js (hunk 1 above)
Comment (platfowner) [reply node/index.js]: ditto

File: node/index.js (hunk 2 above)
Comment (platfowner) [reply node/index.js]: Just done, thanks for the nice idea!

**Comment by platfowner** [comment]
Comments address, PTAL~ @liayoo

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

**Pull request merged**